examples/test-upgrade/v110/cloud-backup-snapshot/v110/main.tf

```diff
@@ -8,7 +8,7 @@ resource "mongodbatlas_project" "project_test" {
 resource "mongodbatlas_cluster" "cluster_test" {
   project_id   = mongodbatlas_project.project_test.id
   name         = var.cluster_name
-  disk_size_gb = 5
+  disk_size_gb = 10
```

Comment:
Since you are changing this anyway can you just remove it?  Having disk size, unless it's bigger than the default, really trips uses up if we give them more storage.